cmd/output/notify.go

```diff
@@ -3,14 +3,18 @@ package output
 import (
 	"fmt"
 	"math/rand"
+	"strings"
 )
 
 var notifyCmdMovedBody = AnsiFaint + "A new set of commands related to apps in organizations have been promoted as \n" +
 	"the default commands, and previous commands have been moved to the \"legacy\" namespace." + AnsiReset + "\n\n" +
 	"See https://www.numerous.com/docs/cli#legacy-commands for more information."
 
 func Notify(header, body string, args ...any) {
-	fmt.Printf(AnsiCyanBold+header+AnsiReset+"\n"+body+"\n", args...)
+	if body != "" && strings.HasSuffix(body, "\n") {
+		body += "\n"
+	}
```

Comment:
Just a minor note. It can be replaced with `body = addTrailingNewLine(body)`.

Reply:
Nice catch :)

Reply:
fixed in 07f37545e642228c22c6c5c4c5812f1ada673dd5